src/docs/cookbook/navigation/navigation-basics.md

```diff
@@ -19,24 +25,21 @@ So how do we navigate to new screens? Using the [`Navigator`](https://docs.flutt
 
 ## 1. Create two screens
 
-First, we'll need two screens to work with. Since this is a basic example, we'll
-create two screens, each containing a single button. Tapping the button on the
-first screen will navigate to the second screen. Tapping the button on the
-second screen will return our user back to the first!
-
-First, we'll set up the visual structure.
+In this basic example, there are two screens, each containing a single button. 
+- Tapping the button on the first screen will navigate to the second screen. 
+- Tapping the button on the second screen will return back to the first screen.
 
 ```dart
 class FirstScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
-        title: Text('First Screen'),
```

Comment:
I don't feel that these code changes really improve the understanding of the code. Also, they should use sentence case so:

title: Text('First Screen'),  =>  title: Text('First screen'),

They are currently inconsistent.